code/modules/admin/view_variables/get_variables.dm

```diff
@@ -11,6 +11,8 @@
 	else if(istext(var_value))
 		if(findtext(var_value, "\n"))
 			. = VV_MESSAGE
+		else if(findtext(var_value, GLOB.is_color))
```

Comment:
I'm not confident this is the best idea, though our admins should be trained to use (C) in the case of any places this would fail anyway.